src/main/java/at/hannibal2/skyhanni/utils/UtilsPatterns.kt

```diff
@@ -41,7 +41,7 @@ object UtilsPatterns {
 
     val baitPattern by patternGroup.pattern(
         "item.name.bait",
-        "^(Obfuscated.*|.* Bait)$",
+        "^(?:Obfuscated.*|.* Bait)$",
```

Comment:
```suggestion
        "^Obfuscated.*|.* Bait$",
```
Will need update in baseline